holmes/plugins/toolsets/bash/bash_toolset.py

```diff
@@ -25,7 +24,7 @@
 
 
 class BaseBashExecutorToolset(Toolset):
-    config: Optional[BashExecutorConfig] = None
+    typed_config: Optional[BashExecutorConfig] = None
```

Comment:
:+1:

Reply:
Instead of reusing config, which might be confused with Toolset.config, I use typed_config for specific toolsets for a typed config.
FYI @arikalon1  We had discussions around this variable naming, but now we are on the same page to use typed_config to be specific.

Reply:
@nherment @mainred Can you share the reason for choosing a different member, instead of overriding the `config` member on the base class (like in an OOP manner)?